translations/translations.json

```diff
@@ -4555,119 +4526,148 @@
   "search": {
     "de": {
       "searchPlaceholder": "Suche",
-      "pageTitle": "Suche"
+      "pageTitle": "Suche",
+      "nothingFound": "Es wurden leider keine passenden Ergebnisse gefunden."
```

Comment:
I think this should go to the error namespace if it's only used in the failure component

Reply:
Not really an error though. I think it's fine to have it here.

Reply:
If it's only used in the `Failure` component it should also be part of the namespace the failure component uses. Also, `search` is the namespace for the category search, not search in general. It also makes the code easier, so I definitely think it should be moved :)

Reply:
The translations in the error.notFound section translated by the FailureSwitcher component, which I don't use. So I can't leave out the explicit namespace without using the FailureSwitcher component.

I prefer using using only the namespace search in the search.
For the city search I agree that the "search" is the wrong content.
I used error - notFound.city now which is the existing message for a wrong spelled city in the url: Diese Stadt wurde leider nicht gefunden.

Reply:
You are absolutely right. However, I think `Es wurden leider keine passenden Ergebnisse gefunden` is the better text for the city search compared to `Diese Stadt wurde leider gefunden`. Reasons:
* We try to avoid the term `Region` as much as possible (`Stadt` should not be used in the future anymore at all in the UI)
* `Keine passenden Ergebnisse` is more general, the user may just have entered `as` which already gives no result but it is obviously not a city

So best just revert the last commit and use the `search` context again (or move the `const { t } = useTranslation('error')` into the `Failure` component and `nothingFound` to the error namespace).